cmake/projects/Libevent/hunter.cmake

```diff
@@ -9,8 +9,8 @@ include(hunter_pick_scheme)
 hunter_add_version(
     PACKAGE_NAME Libevent
     VERSION "2.1.8"
-    URL "https://github.com/hunter-packages/libevent/archive/v2.1.8-p2.tar.gz"
-    SHA1 "492abb962dad3071f34bbe4975e28768fd29edf0")
+    URL "https://github.com/hunter-packages/libevent/archive/v2.1.8-p3.tar.gz"
```

Comment:
Add new `hunter_add_version` instead and update `default.cmake`:
* https://docs.hunter.sh/en/latest/creating-new/update.html

Reply:
OK will do